flocker/node/agents/test/test_blockdevice.py

```diff
@@ -66,6 +68,18 @@
 from ....common.test.test_thread import NonThreadPool, NonReactor
 
 LOOPBACK_BLOCKDEVICE_SIZE = int(MB(64).to_Byte().value)
+CLEANUP_RETRY_LIMIT = 10
+
+# Helper structures to Eliot output leftover volumes after a test run.
+VOLUMES = Field.for_types(
```

Comment:
Thanks a lot for the explanation! Will modify this branch since this is a small change.